documents/Libraries/stdlib/gen-glsl/ogsfx/mx_image_color20.glsl

```diff
@@ -0,0 +1,14 @@
+void mx_image_color2(sampler2D tex_sampler, int layer, vec2 defaultval, vec2 texcoord, int uaddressmode, int vaddressmode, int filtertype, int framerange, int frameoffset, int frameendaction, out vec2 result)
```

Comment:
File name: mx_image_color20.glsl -> mx_image_color2.glsl

Reply:
Or why is there a 0 appended? (I noticed the same pattern on all files below)

Reply:
Seems to be duplicate files.

Reply:
Probably a merge issue. I'll fix it up.